src/sco.c

```diff
@@ -457,7 +459,8 @@ static void *sco_msbc_dec_thread(struct ba_transport_thread *th) {
 		if (!ba_transport_pcm_is_active(t_pcm))
 			continue;
 
-		ffb_seek(&msbc.data, len);
+		if (len > 0)
+			ffb_seek(&msbc.data, len);
```

Comment:
Why msbc thread has got this `if (len>0)` check, where cvsd has got `continue` after read error? I think that the proper fix is to add `continue` after read error. It's definitely a bug to go forward with `-1` as length :D

Reply:
OK, my bad.... indeed we need to feed msbc_decoder so PLC can work properly